3-blank/README.md

```diff
@@ -88,7 +92,7 @@ For the full command list please view the [Docker Documentation](https://docs.do
 
 - [Check frequently asked questions](https://discord-ts.js.org/docs/faq)
 - [Check examples](https://github.com/oceanroleplay/discord.ts/tree/main/packages/discordx/examples)
-- Ask in [Discord Server](https://discord-ts.js.org/discord)
+- Join and Ask in our [Discord Server](https://discord-ts.js.org/discord)
```

Comment:
revert this for all, I found the original one better

Reply:
Ye but it don't make sense... How its currently written leaves `Discord Server` undefined

Reply:
@TeamWolfyta is right. "Ask in Discord server" doesn't make sense

Reply:
the abstract is, ``Ask`` in ``discord server``, point out, ``our`` and `join` does not make much sense.

Reply:
Maybe, ``Ask in Community Discord Server`` would be more accurate.

Reply:
> Maybe, `Ask in Community Discord server` would be more accurate.

`Ask in the community Discord server`

Reply:
ok